src/python/README.md

```diff
@@ -43,6 +43,7 @@ Installing
 -----------------------
 
 - [Install the gRPC core](https://github.com/grpc/grpc/blob/master/INSTALL)
+(n.b. this is not currently pulled in as a dependency!)
```

Comment:
Is it kosher to leave a direct link into github, e.g. just copy the link you've provided directly into the README?